ui/locales/de.json

```diff
@@ -19,7 +19,7 @@
   "FULL_SCREEN": "Vollbildmodus",
   "LANGUAGE": "Sprache",
   "LIVE": "Live",
-  "LOOP": "Schleife",
+  "LOOP": "Wiederholen",
```

Comment:
The previous translation with `Schleife` or `Dauerschleife` (aka endless loop) is better. `Wiederholen` is a word for `repeat`.

Reply:
Vielen Dank!

Reply:
Reverted.